src/pages/AddTransactionForm.jsx

```diff
@@ -41,7 +41,7 @@ export default function AddTransactionForm({fetchTransactions}) {
           addTransactionToDb(newTransaction) 
         }
         else if (today < transactionDate){
-          alert("Never know tomorrow ")
+          alert("That is in the future")
```

Comment:
Also, how far back can they update something? If they are adding something for the previous month, do our reports change/have to be regenerated?

Reply:
I think it's fine to add something to the past, but I think I should put restrictions on it, so I limit it within a year. for example,  we could add 2022 - today's transactions